### PR TITLE
Raj_Assignment_27_50min_Done

### PR DESCRIPTION
**Pull request by RajAJun20**
Assignment_27_MeetingRoom

### CODE REVIEW

**Review by AnshuKJun20 — APPROVED** [review]
Output is as expected.

File: src/rajAJun20/MeetingRoom.java [hunk 1]
```diff
@@ -0,0 +1,110 @@
+package rajAJun20;
+
+public class MeetingRoom {
+	enum ROOMS {
+		NALANDA, TAKSHASHILA, AGRAFORT, PRATPGADH;
+	}
+
+	static boolean nalandaBookingStatus, takshashilaBookingStatus, agraFortBookingStatus, pratapGadhBookingStatus;
+
```
Comment (AnshuKJun20) [inline src/rajAJun20/MeetingRoom.java]: Remove extra space.

**Review by AnshuKJun20 — COMMENTED** [review]
Remove extra space.  Otherwise code looks good.

**Comment by RajAJun20** [comment]
Removed Space

**Review by AnshuKJun20 — APPROVED** [review]

**Pull request merged**